docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Roadmap
+
+The SAPS is currently composed of four main services: `Archiver`, `Catalog`, `Dispatcher` and `Scheduler`. In addition, saps uses a job execution service called `Arrebol`. Given the context, below we have the correct deployment sequence for Saps components.
```

Comment:
SAPS is currently composed... no need for 'THE'

Reply:
Saps uses a job execution service, but it has to be arrebol? or can it be whichever the user wants? If thats the case u could say as arrebol for example.

Reply:
Yes, the execution service can be other than Arrebol. I added in the doc.